tests/load/pipeline/test_csv_loading.py

```diff
@@ -0,0 +1,73 @@
+import os
+import pytest
+
+from dlt.common.utils import uniq_id
+
+from tests.load.pipeline.utils import destinations_configs, DestinationTestConfiguration
+from tests.cases import arrow_table_all_data_types, prepare_shuffled_tables
+from tests.pipeline.utils import assert_data_table_counts, assert_load_info, load_tables_to_dicts
+from tests.utils import TestDataItemFormat
+
+
+@pytest.mark.parametrize(
+    "destination_config",
+    destinations_configs(default_sql_configs=True, subset=["postgres", "snowflake"]),
```

Comment:
we should probably filter the tests by loader_file_format in caps and not explicitely in the test header, then all destinations that support csv will automatically be added to this test.

Reply:
90% of explicit config can be inferred from caps and config interfaces